server/app/views/questiontypes/AddressQuestionRenderer.java

```diff
@@ -48,6 +51,7 @@ protected DivTag renderTag(
                         messages,
                         validationErrors.getOrDefault(
                             addressQuestion.getStreetPath(), ImmutableSet.of()))
+                    .setAriaInvalid(hasErrors)
```

Comment:
Is there a missing call to add the described by IDs here and in other renderers?

Reply:
Oh, I think I had missed that this would be considered a "COMPOSITE" field and the described by would be associated with the top level field? If so, I wonder if we should have separate methods for rendering composite fields versus single fields since the described by IDs would be extraneous here.

I could also see this being a case where we'd add some inheritance in order to make it explicit whether a question type was composite or singular. As it stands, it's a bit hard to follow why the parameter is ignored in some question types and used in others.

Reply:
For the purposes of this PR, I'm definitely up for any of the options. My general feeling is that separate classes (e.g. option #3) might be the most maintainable long-term since it has very clearly defined boundaries and hopefully no unused code / calls.

Reply:
Agreed that (3) sounds like the most correct approach. I'll send out a separate PR to refactor so this one stays smaller.